src/packaging/_manylinux.py

```diff
@@ -14,6 +14,8 @@
 EF_ARM_ABI_FLOAT_HARD = 0x00000400
 
 
+# `os.PathLike` not a generic type until Python 3.9, so sticking with `str`
+# as the type for `path` until then.
```

Comment:
This seems unrelated to the metadata change.

Reply:
It is, but it's minor enough I didn't want to bother with a separate PR.